site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -418,6 +420,69 @@
             }
             e.stopPropagation();
         });
+
+        var serverDate;
+
+        var fetchTimeUrl = window.location.href + "&action=fetch_time";
+
+        var nextTimeFetch = 0 ;
+        var setTimerOn = 0;
+
+        fetchServerTime();
+
+
+        function fetchServerTime() {
+
+            var localDate = new Date();
+
+            console.log("Fetching Server");
+            if(nextTimeFetch == 0) {
+                // console.log("Fetch set time remaining: "+(60 - localDate.getSeconds()));
+                nextTimeFetch = 300;
+                setTimeout(function(){
+                    setInterval(fetchServerTime, nextTimeFetch * 1000);
+                    // console.log("Next fetch set");
+                }, (60 - localDate.getSeconds())*1000);
+            }
+
+            $.ajax({
```

Comment:
Could you remove the console.logs and also the commented out ones and replace them with comments about the logic of the function, I think that would be more helpful